src/main/java/sonaKh/homework_1/TheSumAndTheAverageOfTheNumbers.java

```diff
@@ -0,0 +1,41 @@
+package sonaKh.homework_1;
+
+import java.util.Scanner;
+
+public class TheSumAndTheAverageOfTheNumbers {
+
+    private static void theSumAndTheAverage(int number1, int number2) {
+        int sum = 0;
+        double average = 0;
+        if (number1 < number2) {
+            average = number2 - number1;
```

Comment:
obshi kashxati, bayc optimal chi karas senc anes:
average = Math.abs(number2 - number1); sa kvekali bacardzak arjeqy u sa heriqa qez, el petq chi erku hat if u for gres :)